web/styles/popovers.css

```diff
@@ -856,3 +856,18 @@ ul {
         column-count: 1;
     }
 }
+
+#send_later_popover {
+    & hr {
+        margin: 5px 0;
+    }
+
+    .selected::before {
+        content: attr(data-before);
+        position: absolute;
+        width: 102px;
```

Comment:
This very specific width makes me worry a bit about this not handling i18n well.

Reply:
(No longer relevant since we removed the selected style completely.)